webdevnerdstuff/vuetify-drilldown-table
# v1.1.0

* Add vitest
* Fix typescript issues
* Reorganizing
* Fix issue with header color not working on sortable columns
* Add level check to fix opacity issue
* Update node version to v20.10.0
